packages/amplify_auth_plugin_interface/lib/amplify_auth_plugin_interface.dart

```diff
@@ -91,4 +93,23 @@ abstract class AuthPluginInterface extends AmplifyPluginInterface {
   Future<SignInResult> signInWithWebUI({SignInWithWebUIRequest request}) {
     throw UnimplementedError('signInWithWebUI() has not been implemented.');
   }
+
+  Future<UpdateUserAttributeResult> updateUserAttribute(
```

Comment:
I think we should have consistent formatting on all methods in this interface file.

Reply:
I removed the trailing comma to be consistent with the file/project, but left the formatting from dartfmt on the newly added methods. I think we *should* be using dartfmt if we aren't.